packages/esm-patient-chart-app/src/mark-patient-deceased/mark-patient-deceased-form.workspace.tsx

```diff
@@ -134,22 +139,13 @@ const MarkPatientDeceasedForm: React.FC<DefaultPatientWorkspaceProps> = ({ close
                   name="deathDate"
                   control={control}
                   render={({ field: { onChange, value } }) => (
-                    <DatePicker
+                    <OpenmrsDatePicker
                       className={styles.datePicker}
-                      dateFormat="d/m/Y"
-                      datePickerType="single"
                       id="deceasedDate"
```

Comment:
We need to maintain parity between the props passed to the Carbon datepicker and those passed to `OpenmrsDatePicker`. In this case, this implementation is missing several props including `labelText`, `isInvalid`, and `invalidText`. It should look like this instead:

```tsx
<OpenmrsDatePicker
    className={styles.datePicker}
    id="deceasedDate"
    isInvalid={!!errors?.deathDate}
    invalidText={errors?.deathDate?.message}
    labelText={t('dateOfDeath', 'Date of death')}
    maxDate={new Date().toISOString()}
    onChange={(date) => onChange(date)}
    value={value}
  />
```